spec/helpers/pages_helper_spec.rb

```diff
@@ -46,6 +46,21 @@
     end
   end
 
+
+  describe 'ak_resource_id' do
+
+    it 'parses an ak petition resource uri to an ID' do
+      ak_resource_url = 'https://act.sumofus.org/rest/v1/petitionpage/11207/'
+      expect(helper.ak_resource_id(ak_resource_url)).to eq '11207'
+    end
+
+    it 'parses an ak donation resource uri to an ID' do
+      ak_resource_url = 'https://act.sumofus.org/rest/v1/donationpage/12345/'
+      expect(helper.ak_resource_id(ak_resource_url)).to eq '12345'
+    end
+
```

Comment:
would be good for a test case with no trailing slash. Also you've got rubocop problems :roll_eyes: